build_debian.sh

```diff
@@ -524,6 +524,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'docke
 # Install scapy
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'scapy==2.4.4'
 
+# Install grpcio
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'grpcio==1.58.0'
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'grpcio-tools==1.58.0'
```

Comment:
These packages are already being installed in `files/build_templates/sonic_debian_extension.j2`.

Reply:
@saiarcot895 , have removed it from here and added it to sonic_debian_extension.j2

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/18536#discussion_r1766183415